<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>AddVariable</class>
 <widget class="QDialog" name="AddVariable">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>413</width>
    <height>310</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>添加变量</string>
  </property>
  <widget class="QWidget" name="verticalLayoutWidget">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>411</width>
     <height>311</height>
    </rect>
   </property>
   <layout class="QVBoxLayout" name="verticalLayout">
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout">
      <item>
       <widget class="QLabel" name="label">
        <property name="minimumSize">
         <size>
          <width>100</width>
          <height>0</height>
         </size>
        </property>
        <property name="layoutDirection">
         <enum>Qt::LeftToRight</enum>
        </property>
        <property name="autoFillBackground">
         <bool>false</bool>
        </property>
        <property name="styleSheet">
         <string notr="true">font: 12pt &quot;华文楷体&quot;;</string>
        </property>
        <property name="text">
         <string>变量名称</string>
        </property>
        <property name="alignment">
         <set>Qt::AlignCenter</set>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLineEdit" name="variable_name">
        <property name="enabled">
         <bool>true</bool>
        </property>
        <property name="maxLength">
         <number>10</number>
        </property>
        <property name="frame">
         <bool>true</bool>
        </property>
       </widget>
      </item>
     </layout>
    </item>
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout_3">
      <item>
       <widget class="QLabel" name="label_2">
        <property name="styleSheet">
         <string notr="true">font: 12pt &quot;华文楷体&quot;;</string>
        </property>
        <property name="text">
         <string>变量类型</string>
        </property>
        <property name="scaledContents">
         <bool>true</bool>
        </property>
        <property name="alignment">
         <set>Qt::AlignCenter</set>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QRadioButton" name="type_number">
        <property name="styleSheet">
         <string notr="true">font: 12pt &quot;华文楷体&quot;;</string>
        </property>
        <property name="text">
         <string>数字</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QRadioButton" name="type_matrix">
        <property name="styleSheet">
         <string notr="true">font: 12pt &quot;华文楷体&quot;;</string>
        </property>
        <property name="text">
         <string>矩阵</string>
        </property>
       </widget>
      </item>
     </layout>
    </item>
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout_2">
      <item>
       <widget class="QLabel" name="label_3">
        <property name="styleSheet">
         <string notr="true">font: 12pt &quot;华文楷体&quot;;</string>
        </property>
        <property name="text">
         <string>数域</string>
        </property>
        <property name="textFormat">
         <enum>Qt::AutoText</enum>
        </property>
        <property name="scaledContents">
         <bool>true</bool>
        </property>
        <property name="alignment">
         <set>Qt::AlignCenter</set>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QRadioButton" name="num_Q">
        <property name="styleSheet">
         <string notr="true">font: 12pt &quot;华文楷体&quot;;</string>
        </property>
        <property name="text">
         <string>有理数Q</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QRadioButton" name="num_R">
        <property name="styleSheet">
         <string notr="true">font: 12pt &quot;华文楷体&quot;;</string>
        </property>
        <property name="text">
         <string>实数R</string>
        </property>
       </widget>
      </item>
     </layout>
    </item>
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout_5">
      <item>
       <widget class="QLabel" name="label_4">
        <property name="styleSheet">
         <string notr="true">font: 12pt &quot;华文楷体&quot;;</string>
        </property>
        <property name="text">
         <string>数值（仅数字可赋值）</string>
        </property>
        <property name="textFormat">
         <enum>Qt::AutoText</enum>
        </property>
        <property name="scaledContents">
         <bool>true</bool>
        </property>
        <property name="alignment">
         <set>Qt::AlignCenter</set>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLineEdit" name="variable_value">
        <property name="enabled">
         <bool>true</bool>
        </property>
       </widget>
      </item>
     </layout>
    </item>
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout_6" stretch="0,0,0,0,0">
      <item>
       <widget class="QLabel" name="label_5">
        <property name="styleSheet">
         <string notr="true">font: 12pt &quot;华文楷体&quot;;</string>
        </property>
        <property name="text">
         <string>矩阵大小</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLineEdit" name="Row"/>
      </item>
      <item>
       <widget class="QLabel" name="label_6">
        <property name="styleSheet">
         <string notr="true">font: 12pt &quot;华文楷体&quot;;</string>
        </property>
        <property name="text">
         <string>行</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLineEdit" name="Column"/>
      </item>
      <item>
       <widget class="QLabel" name="label_7">
        <property name="styleSheet">
         <string notr="true">font: 12pt &quot;华文楷体&quot;;</string>
        </property>
        <property name="text">
         <string>列</string>
        </property>
       </widget>
      </item>
     </layout>
    </item>
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout_4">
      <property name="leftMargin">
       <number>100</number>
      </property>
      <item>
       <widget class="QPushButton" name="confirm">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Minimum" vsizetype="Fixed">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="maximumSize">
         <size>
          <width>100</width>
          <height>16777215</height>
         </size>
        </property>
        <property name="styleSheet">
         <string notr="true">font: 12pt &quot;华文楷体&quot;;</string>
        </property>
        <property name="text">
         <string>确认</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="cancel">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Minimum" vsizetype="Fixed">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="maximumSize">
         <size>
          <width>100</width>
          <height>16777215</height>
         </size>
        </property>
        <property name="styleSheet">
         <string notr="true">font: 12pt &quot;华文楷体&quot;;</string>
        </property>
        <property name="text">
         <string>取消</string>
        </property>
        <property name="default">
         <bool>false</bool>
        </property>
        <property name="flat">
         <bool>false</bool>
        </property>
       </widget>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
